modules/network-observability-overview.adoc

```diff
@@ -0,0 +1,22 @@
+= Using Overview
+Overview displays the overall, aggregated metrics of the network traffic flow on the cluster. As an administrator, you can monitor the statistics with the available display options.
+
+== Working with Overview
+As an administrator, you can navigate to overview by using the following steps:
+
+. On the left navigation panel, click *Observe* → *Network Traffic*.
+. In the *Network Traffic* page, click the *Overview* tab.
+
+You can view the graphical representation of the flow rate statistics. You can configure the scope of each flow rate data by clicking the menu icon.
+
+== Configuring advanced options
+You can customize the graphical view by using advanced options. To access the advanced options, click *Show advanced options*.
+
+You can configure the details in the graph by using the *Display options* dropdown. The options available are:
+
+* *Metric type*: The metrics to be shown in graph. The default value is *Bytes*.
+* *Scope*: To choose the detail of components between which the network traffic flows. The default value is *Namespace*.
```

Comment:
I would also mention what is available

```suggestion
* *Scope*: To choose the detail of components between which the network traffic flows: *Node*, *Namespace*, *Owner* or *Resource*. The default value is *Namespace*.
```

Maybe "Owner" and "Resource" deserve a bit more explanation. I give you it in my words, which probably aren't 100% ok for doc:

- Resource is the "leaf" resource (probably not a good wording for doc) and  can designate a Pod , a Service, a Node or an unknown (unmatched) IP address.
- Owner, a.k.a Workload, designates an aggregation of resources, and can typically be a Deployment, a StatefulSet, a DaemonSet, etc. (there's no exhaustive list)